webui/src/pages/user/add-namespace-member-dialog.tsx

```diff
@@ -124,11 +124,10 @@ export const AddMemberDialog: FunctionComponent<AddMemberDialoProps> = props =>
                     label='Open VSX User'
                     fullWidth
                     onChange={handleUserSearch}
-                    onKeyPress={(e: React.KeyboardEvent) => {
```

Comment:
Removed the explicit type for the event here - we get it from the component and there's no need for us to do this.

Reply:
It's TypeScript. The webui code is riddled with React events. Let's keep things consistent.